components/tidb_query/src/rpn_expr/impl_json.rs

```diff
@@ -187,6 +191,27 @@ fn json_extract(args: &[ScalarValueRef]) -> Result<Option<Json>> {
     Ok(j.extract(&path_expr_list))
 }
 
+// Args should be like `(&Option<Json> , &[&Option<Bytes>])`.
+fn json_with_path_validator(expr: &tipb::Expr) -> Result<()> {
+    assert!(expr.get_children().len() == 2 || expr.get_children().len() == 1);
+    valid_paths(expr)
+}
+
+#[rpn_fn(raw_varg, min_args = 2, extra_validator = json_with_path_validator)]
+#[inline]
+fn json_length(args: &[ScalarValueRef]) -> Result<Option<Int>> {
```

Comment:
Maybe you need to add `max_args` to avoid TiKV panic when wrong number of arguments is passed?

Reply:
Yes，I can add max_args to rpn_fn. so I need to open new pr to implement this.

Reply:
Done